src/components/course/course-header/data/hooks/useCourseRunCardAction.jsx

```diff
@@ -54,6 +54,7 @@ const useCourseRunCardAction = ({
     handleRedeemSuccess,
     handleRedeemError,
   } = useRedemptionStatus();
+  const useExternalCourseEnrollmentUrlForCourseware = isUserEnrolled && externalCourseEnrollmentUrl;
```

Comment:
nit: perhaps `hasExternalCourseEnrollmentUrlForCourseware` or similar? variables starting with `use` are usually reserved as React hooks, which could be misleading.

Reply:
Good call-out!